merino-suggest/src/domain.rs

```diff
@@ -39,6 +39,7 @@ impl Proportion {
     }
 }
 
+/// Define helper impls for float values
```

Comment:
heads up: I think this might conflict with some other PR, but I forget which one.